modules/dynatrace-private-activegate/vm.tf

```diff
@@ -0,0 +1,82 @@
+locals {
+  prefix    = "activegate-private-${var.env}"
+  adminuser = "azureuser"
+}
+
+data "azurerm_subnet" "iaas" {
+  name                 = "iaas"
+  resource_group_name  = var.resource_group_name
+  virtual_network_name = var.vnet_name
+}
+
+data "azurerm_key_vault" "subscription_vault" {
+  name                = var.vault_name
+  resource_group_name = var.vault_rg
+}
+
+data "azurerm_key_vault_secret" "ssh_public_key" {
+  name         = "aks-ssh-pub-key"
+  key_vault_id = data.azurerm_key_vault.subscription_vault.id
+}
+
+
+data "template_file" "cloudconfig" {
+  template = file("${path.module}/cloudconfig.tpl")
+
+  vars = {
+    paas_token               = data.azurerm_key_vault_secret.dynatrace_paas_token.value
+    dynatrace_instance_name  = var.dynatrace_instance_name
+    network_zone             = var.network_zone
+  }
+}
+
+data "template_cloudinit_config" "config" {
+  gzip          = true
+  base64_encode = true
+
+  part {
+    content_type = "text/cloud-config"
+    content      = data.template_file.cloudconfig.rendered
+  }
+}
+
+resource "azurerm_linux_virtual_machine_scale_set" "main" {
```

Comment:
there a reason we don't have log in with AAD enabled here? is it missing in the other active gate?

Reply:
Yup, direct copy from the other active gate module